kafka-ui-api/src/main/java/com/provectus/kafka/ui/util/KafkaVersion.java

```diff
@@ -1,24 +1,21 @@
 package com.provectus.kafka.ui.util;
 
-import lombok.extern.slf4j.Slf4j;
+import java.util.Optional;
 
-@Slf4j
 public final class KafkaVersion {
 
   private KafkaVersion() {
   }
 
-  public static float parse(String version) throws NumberFormatException {
-    log.trace("Parsing cluster version [{}]", version);
+  public static Optional<Float> parse(String version) throws NumberFormatException {
```

Comment:
it doesn't seem to throw declared exception, as all thrown ones should be catched instead. Could you please double-check that ?